static/js/chart/draw_bar.ts

```diff
@@ -326,14 +326,16 @@ export function drawStackBarChart(
   const colorFn = getColorFn(colorOrder, options?.colors);
 
   const setData = (d: d3.Series<{ [key: string]: number }, string>) => {
-    return d.map((item) => ({
-      date: item.data.date,
-      place: item.data.label,
-      statVar: d.key,
-      unit: options?.unit,
-      value: item.data.value,
-      ...item,
-    }));
+    return d
+      .filter((item) => !isNaN(item[0]) && !isNaN(item[1]))
```

Comment:
curious, will item always be an array with at least 2 values? maybe leave a comment about this?

Reply:
+1
Would it make sense to add a bounds check here and filter out the items if the size != 2?

Reply:
Yes, item will always have at least 2 items. It's a `d3.SeriesPoint` with the weird syntax [elem1, elem2, data:{...}]. I think adding a bounds check is a great idea though. Added to both instances of .filter().